.github/workflows/code-sanity-on-push.yml

```diff
@@ -0,0 +1,19 @@
+name: Code Sanity
+on: 
+  push:
+    branches:
+      - 'issue*'
+      - 'feature*'
```

Comment:
don't we wanna do same checks for the master branch?

Reply:
Indeed we should do a complete set of tests when merging into master branch.

As I am not sure how github actions allows multiple workflow for a single set of condition, I am working on a test repository to see how to make a complete implementation.

I thought to implement something like (at least):

- Push to `issue*` or `feature*`: 1 Workflow for code linting
- Merge request to `*-release`: 1 worklow with code linting + `ansible-test sanity --test import`
- Merge to `master`: Release management (tag), docker build & push, ansible-galaxy stuff

Meaning that if we do a CI for the first 2 steps, linting should not be required when reaching `master` as code would have been reviewed at least 2 times.

Does it make sense for you ?

Reply:
yep, makes sense, i had a similar idea in mind.

Reply:
Here is an example implementing pylint for pushes in any `feature*` or `issue*`

For PR, GA runs a full ansible-test set of actions using docker

https://github.com/titom73/ansible-test/actions

For `master`, it will be created later